build.ps1

```diff
@@ -74,7 +74,7 @@ if (Test-Path "$RepoRoot\artifacts")
 
 mkdir "$RepoRoot\artifacts" | Out-Null
 
-$DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1"
+$DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1"
```

Comment:
This is what I wasn't sure about. They seem to be able to update to 2.0, with no changes to their project structure. Tests seem to pass, is there some other factor that is blocking the update?